tests/test_commands.py

```diff
@@ -0,0 +1,69 @@
+import subprocess
+import sys
+
+from .test_plugins.test_plugins import function_like_plugin
+
+try:
+    import importlib.metadata as importlib_metadata
+except ModuleNotFoundError:
+    import importlib_metadata
+
```

Comment:
You don't need to import `importlib.metadata`, rather you can import `__version__` from manim.

Reply:
I will say that is fine to do because I am testing whether the `__version__` works or not. (I know it will always but in case)
https://github.com/ManimCommunity/manim/blob/0b3d39586ceac7c6fec953c730a7702a8ae7c3f8/tests/test_commands.py#L39

Reply:
If you know that something always has to be true, it seems to me like it would add clutter to test.

Reply:
removed :)